### ISSUE TITLE
how does a first-timer get to know about activity streams 

### ISSUE THREAD

**Issue by MohanL** [exercism/exercism.io#3220]
![image](https://cloud.githubusercontent.com/assets/10900547/19839517/e2dbc7f2-9eb9-11e6-9de9-fd84b7a75c40.png)

For example, Alice won't know about the activity streams until she turns in her first submission.

**Comment by Insti** [comment]
Making it always visible is a good idea :+1:

**Comment by jtigger** [comment]
Huh... I'm not clear as to why this is a good idea.  If a user has not yet submitted a solution, there's nothing to see.  In fact, they are more likely lost than anything else.  Providing an action that takes them away from what should be their next step (i.e. submitting their first solution) seems to create more confusion.

What if, instead, there is a persistent flash/notice that says, "The next goal is to submit your first solution; start here." where "here" links to http://exercism.io/how-it-works/newbie ?

**Comment by jtigger** [comment]
To the spirit of this issue, we could have a call to action on the Solution page, when the user is viewing their own solution, to visit the Activity Stream.

**Comment by kytrinyx** [comment]
> we could have a call to action on the Solution page, when the user is viewing their own solution, to visit the Activity Stream

We kind of have this.

<img width="439" alt="go-to-activity-stream" src="https://cloud.githubusercontent.com/assets/276834/20032105/22402c88-a349-11e6-9d91-d26b8e14db7c.png">

So the thing that I want to avoid is if someone gets a link to the activity stream but they haven't done anything, they end up on a page that gives them an error or just has nothing on it.

I think the thing we need is:

* We don't link to the activity stream in the nav if you haven't submitted anything, but
* If you happen to go to the activity stream and you don't have any activity yet, we should show you an explanation: _Submit a solution to an exercise, and you'll get access to all the other solutions to the same exercise here. We'll also link to this in the top navigation._ Except written more diplomatically.
* If you happen to go to a specific activity stream that you don't have access to, we should say "you can see this when you've submitted a solution to it". And then maybe show them the `exercism fetch` command that would give them that exercise.

Or something.

**Comment by HarrisonMc555** [comment]
I agree with @kytrinyx on this issue. It took me a while before I realized that I wasn't "allowed" to see solutions for exercises I hadn't submitted anything to.

I also agree with @jtigger that we should avoid bringing up the Activity Stream until after they've submitted their first solution. I would have appreciated a one-time banner or header or something that explained the concept of the "Activity Streams".
